client/src/components/App/index.js

```diff
@@ -1,13 +1,16 @@
 import React from 'react';
 import Header from '../common/Header';
 import Footer from '../common/Footer';
+import LogIn from '../pages/LogIn';
 import './index.css';
 
 function App() {
   return (
     <>
       <Header />
-      <main className="container"></main>
+      <main className="container">
+        <LogIn />
```

Comment:
you should to remove Component here and above in import
you make test by it them remove it